pkgs/top-level/all-packages.nix

```diff
@@ -2241,6 +2241,8 @@ in
 
   socklog = callPackage ../tools/system/socklog { };
 
+  spacevim = callpackage ../applications/editors/spacevim {};
```

Comment:
As the bot shows this doesn't build because you made a typo here. It's supposed to be `callPackage` with a capital P and I would also but a space between the curly braces.

Reply:
```suggestion
  spacevim = callPackage ../applications/editors/spacevim { };
```